app/src/test/java/com/google/android/ground/ui/datacollection/tasks/number/NumberTaskFragmentTest.kt

```diff
@@ -72,6 +72,15 @@ class NumberTaskFragmentTest : BaseTaskFragmentTest<NumberTaskFragment, NumberTa
     hasValue(NumberTaskData("123.1"))
   }
 
+  @Test
+  fun testResponse_onUserInput_nextButtonIsNotEnabled() = runWithTestDispatcher {
```

Comment:
Let's rename this test to something more appropriate.

`non numeral response gets converted to null`